rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -212,21 +212,18 @@ def train(
                 )
 
     def process(
-        self,
-        message: Message,
-        *,
-        tf_hub_module: Any = None,
-        attribute: Text = TEXT,
-        **kwargs: Any,
+        self, message: Message, *, tf_hub_module: Any = None, **kwargs: Any,
     ) -> None:
-        if attribute not in DENSE_FEATURIZABLE_ATTRIBUTES:
-            return
 
-        sequence_features, sentence_features = self._compute_features(
-            [message], tf_hub_module, attribute=attribute
-        )
+        for attribute in DENSE_FEATURIZABLE_ATTRIBUTES:
```

Comment:
I think it could make sense to add function `get_featurizable_attributes` to the parent class `Featurizer` and then overwrite it in `DenseFeaturizer` and `SparseFeaturizer`. This way the solution is DRY

Reply:
I looked at it and it doesn't really make sense to have it for SparseFeaturizers, as each of them has a different set of featurize they go through. So it would only be for the `DenseFeaturizer` which boils down to the same thing, `get_featurizable_attributes` will  just return `DENSE_FEATURIZABLE_ATTRIBUTES`.